services/cd-service/pkg/service/deploy.go

```diff
@@ -97,7 +99,21 @@ func (d *DeployServiceServer) ReleaseTrain(
 	if err != nil {
 		return nil, internalError(ctx, err)
 	}
-	return &emptypb.Empty{}, nil
+
+	configs, err := d.Repository.State().GetEnvironmentConfigs()
+	if err != nil {
+		return &api.ReleaseTrainResponse{}, err
```

Comment:
please wrap the error here to give context. See function `wrapFileError` as comparison.